docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -168,37 +168,212 @@ Make up your mind on which approach you want to use.
 Want to start a PostgreSQL server on the side with Docker?
 
 [source,shell]
---
+----
 docker run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 \
            --name postgres-quarkus-hibernate -e POSTGRES_USER=hibernate \
            -e POSTGRES_PASSWORD=hibernate -e POSTGRES_DB=hibernate_db \
            -p 5432:5432 postgres:10.5
---
+----
 
 This will start a non-durable empty database: ideal for a quick experiment!
 ====
 
+[[multiple-persistence-units]]
+=== Multiple persistence units
+
+==== Setting up multiple persistence units
+
+[WARNING]
+====
+This feature is still experimental.
+It is considered stable and usable in production but the configuration properties and CDI integration might evolve depending on your feedback.
+
+If you are using this feature, any feedback (even just to say it fulfills your requirements) is very welcome on our link:{quarkus-mailing-list-index}[quarkus-dev mailing list].
+====
+
+Starting with Quarkus 1.7, you can define multiple persistence units using the Quarkus configuration properties approach.
```

Comment:
```suggestion
Starting with Quarkus 1.7.0.Final, you can define multiple persistence units using the Quarkus configuration properties approach.
```

Reply:
It should have been 1.8 so I fixed and I prefer keeping it this way (especially since CR1 already has it).